package/libpiksi/libpiksi/src/util.c

```diff
@@ -97,6 +98,90 @@ bool device_is_duro(void)
                  strlen(DEVICE_DURO_ID_STRING)) == 0);
 }
 
+#define GPS_TIME_FILE_NAME "/var/run/health/gps_time_available"
```

Comment:
Please keep constants at the top of the file

Reply:
fixed